includes/class-sensei-analysis.php

```diff
@@ -45,9 +45,80 @@ public function __construct( $file ) {
 			add_action( 'admin_init', array( $this, 'report_download_page' ) );
 
 			add_filter( 'user_search_columns', array( $this, 'user_search_columns_filter' ), 10, 3 );
+
+			// Add custom navigation.
+			add_action( 'in_admin_header', [ $this, 'add_custom_navigation' ] );
+		}
+	}
+
+	/**
+	 * Add custom navigation to the admin pages.
+	 *
+	 * @since 4.0.0
```

Comment:
Should be 4.2.0?

Reply:
Thanks! Copy-paste error 🤦 
Fixed here: https://github.com/Automattic/sensei/pull/4868/commits/47ac26d121595b60120a01599f66c38f853ca554